packages/ListBox/stories/ListBox.mdx

```diff
@@ -0,0 +1,291 @@
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+import ListBox from "../src/";
+
+import { ImageOption } from "./stories.styles";
+import { images } from "./fixtures/images";
+import * as characters from "./fixtures/characters";
+
+import * as Multi from "./examples/multi";
+import * as Single from "./examples/single";
+import FilterSelect from "./moreExamples/filterSelect/FilterSelect";
+
+# ListBox
+
+The ListBox component allows users to select one or more options from a list contained within the component.
+
+Main features:
+
+- Ability to create custom list items.
+- Ability to make singular or multiple item selections.
+- Accessibility baked in:
+    - Support for screen reader
+    - Support for keyboard navigation ↑, ↓
+    - API for keyboard events Enter and Space.
+
+## Installation
+
+```
+yarn add @/paprika/listbox
+```
+or
+```
+npm install @paprika/listbox
+```
+
+## Usage
+
+### Basic Single ListBox
+
+The single ListBox creates a basic ListBox in which a user can make one selection at a time.
+
+```js
+    <ListBox></ListBox>
+```
+
+To add new list options simply add the child element ``` <ListBox.option></ListBox.option>``` to your ListBox.
+
+```js
+    <ListBox>
+      <ListBox.Option>Option 1</ListBox.Option>
+      <ListBox.Option>Option 2</ListBox.Option>
+    </ListBox>
```

Comment:
Can we make sure all the code examples have the same indentation? I see some inconsistencies with the code blocks